### DISCUSSION TITLE
2.0rc4 The "little" update - is out!

### DISCUSSION THREAD

**Discussion by Gissio** [Gissio/radpro#70] [Announcements]
2.0rc4 is out!

There are new instantaneous rate averaging options, which let you favor precision or fast response. Please read the [updated user's manual](https://github.com/Gissio/radpro/blob/main/docs/users.md#instantaneous-rate-averaging) for learning how it works.

Haptic pulses got an extra boost, and quiet pulse clicks on the FNIRSI GC-01 should be a bit louder now.

Please let us know if display flashes are working correctly on the FNIRSI GC-01.

There is also a new secondary view that displays CPM and counts.

I've also updated the prepackaged [GeigerLog for Rad Pro](https://github.com/Gissio/geigerlog-radpro) to 1.4.3d.

Please share your comments.

**Comment by dc1rdb** [comment]
Haptic pulses are working great now on my GC-01, even at weak setting. Display flashes are working fine, too.
I like the new secondary view displaying CPM.
Have not had a chance to try the new averaging options, will report back when done.

Once again, thanks for your efforts :)

**Comment by Montecri** [comment]
How not to love this project?

**Comment by Gissio** [comment]
One request! Can you post images or videos (YouTube) of your GC-01 in action? Can I use these images/vidoes to showcase Rad Pro?

**Reply by dc1rdb** [reply]
![IMG_20240428_140419~2](https://github.com/Gissio/radpro/assets/9314230/0cc9618b-0be1-4b6b-a24c-8a5a55036936)

https://github.com/Gissio/radpro/assets/9314230/29e2ebf3-7db8-4654-98bd-9604841251d8

Feel free to use them :)

**Reply by Elfe12** [reply]
![image](https://github.com/Gissio/radpro/assets/155153385/ecd52491-706d-46ae-8216-68a7e97cc5ae)

**Comment by Elfe12** [comment]
Feel free to use it. Thank you for this great project. Am 28.04.2024 um 05:50 schrieb Gissio ***@***.***>:﻿
One request! Can you post images or videos of your GC-01 in action? Can I use these images/vidoes to showcase Rad Pro?

**Reply by Gissio** [reply]
[Elfe12](https://github.com/Elfe12): it seems attaches don't make it through email. Could you post again through the web?

**Comment by gkulka-web** [comment]
Can you check "Statistic" page. Tube dead time is shown as 28286us~ 0,03s. I'm not sure how its calculated. Secondly number itself should be in "human friendly" units like hours.

**Reply by Gissio** [reply]
Please read the manual: https://github.com/Gissio/radpro/blob/main/docs/users.md#dead-time-and-dead-time-compensation

Regarding your second comment: dead time is typically 50-200 µs. That's why the value is indicated in µs rather than in ms or s.
